gandi_tf/main.py

```diff
@@ -87,33 +148,71 @@ def generate_tf(domain, entries):
         f.write("  type   = each.value.type\n")
         f.write("  values = each.value.values\n")
         f.write("}\n")
-
+    if subdir is True:
+        with click.open_file(filename_tfimport, "w") as fi:
+            for cmd in commands:
+                fi.write(f"{cmd}\n")
     return commands
 
 
 @click.command(no_args_is_help=True)
 @click.option("--version", help="Display version information", is_flag=True)
+@click.option("--organization_id", help="Gandi organization ID (passing an organization ID will limit query on domain own by organization)", default=None)
+@click.option("--list_domains", help="Fetch and display fqdn_unicode", is_flag=True)
+@click.option("--list_domains_details", help="Fetch and display all domain details", is_flag=True)
+@click.option("--list_organizations", help="Fetch and display all organizations details", is_flag=True)
+@click.option('--nsfilters', help="Filter domaine based on their current nameserver 'abc','livedns' or 'other'. You can add multiple filter Example: gandi2tf --list_domains --nsfilters abc --nsfilters livedns)", multiple=True, default=[
+'abc','livedns','other'])
+@click.option("--auto_generate", help="Auto-generate tf resource based on domain from gandi instead of STDIN/USERINPUT (option can be used with --organization_id, --subdir and multiple --nsfilters)", is_flag=True)
```

Comment:
```suggestion
@click.option("--auto-enerate", help="Auto-generate tf resources based on domain from gandi instead of STDIN/USERINPUT (option can be used with --organization-id, --subdir and multiple --nsfilters)", is_flag=True)
```

Reply:
Hi @marcaurele,

Done renamed auto-generate